megamek/src/megamek/common/Tank.java

```diff
@@ -646,6 +650,12 @@ public void stunCrew() {
     @Override
     public void applyDamage() {
         m_bImmobile |= m_bImmobileHit;
+        //Towed trailers need to use the values of the tractor, or they return Immobile due to 0 MP...
+        if (isTrailer() && getTractor() != null && getTractor().hasETypeFlag(Entity.ETYPE_TANK)) {
+            Tank Tractor = (Tank) getTractor();
+            m_bImmobile = Tractor.m_bImmobile;
+            m_bImmobileHit= Tractor.m_bImmobileHit;
```

Comment:
minor formatting issue with spacing around the `=`

Reply:
Fixed.